stable/pomerium/values.yaml

```diff
@@ -0,0 +1,111 @@
+# For detailed explanation of each of the configuration settings see
+# https://www.pomerium.io/docs/config-reference.htmls
+
+# settings that are shared by all services
+config:
+  # secrets can be generated with `head -c32 /dev/urandom | base64`
+  sharedSecret: "LL3g/NoKkao7mTec2/wYSOZAki1VbR89q+kqbdFj0c4="
+  cookieSecret: "tAhd5A5Y8R/H/msFhkNdt8dCym2SatVbh4UrcDk3+Ms="
+  # default is to use a self-signed cert, should _NOT_ be used inproduction
+  key: "LS0tLS1CRUdJTiBFQyBQUklWQVRFIEtFWS0tLS0tCk1IY0NBUUVFSUtzZ3pUNDdBQXlzTkJNa1BJYnVJV1BodFEwN2RzWGx4WjB5Z3pjV3VXaVVvQW9HQ0NxR1NNNDkKQXdFSG9VUURRZ0FFLzMyY2c0YWpuWHpyT2pORjlXczROK2NyUTdwTGVWOGt6Y3R6cGN1S2F1KzAzVmZXRlRaTQpaTmh2cFQrUzZlS3QzSW1ycExsRTlKRFN6dDZwcUI0UVZRPT0KLS0tLS1FTkQgRUMgUFJJVkFURSBLRVktLS0tLQo="
+  cert: "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJFekNCdWdJSkFLK3VPcmsyVzFRTk1Bb0dDQ3FHU000OUJBTUNNQkV4RHpBTkJnTlZCQU1NQm5WdWRYTmwKWkRBZ0Z3MHhPVEF6TWpBeU1UTTNNemRhR0E4ME56VTNNREl4TkRJeE16Y3pOMW93RVRFUE1BMEdBMVVFQXd3RwpkVzUxYzJWa01Ga3dFd1lIS29aSXpqMENBUVlJS29aSXpqMERBUWNEUWdBRS8zMmNnNGFqblh6ck9qTkY5V3M0Ck4rY3JRN3BMZVY4a3pjdHpwY3VLYXUrMDNWZldGVFpNWk5odnBUK1M2ZUt0M0ltcnBMbEU5SkRTenQ2cHFCNFEKVlRBS0JnZ3Foa2pPUFFRREFnTklBREJGQWlFQTQwSjYycmJjcThYK0EzdTdQemRBNjRhaTJFbWhMQzZkRERYaQpwNmZpaHhFQ0lDRGFWcVkrUHFmZG5Ja1ZQVTFWdU5zNDl5RkJyd2IxYTlqcUdWQzlLaG1GCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K"
+  policyFile: |-
```

Comment:
@desimone same here, if I want to have my own `ConfigMap`, it could be neat to be able to specify a `ConfigMap` name instead of its content.